### DISCUSSION TITLE
QTabs scroll with arrow click should jump to end of next tab

### DISCUSSION THREAD

**Discussion by maggie44** [quasarframework/quasar#13398] [Ideas / Proposals]
When using QTabs and clicking a left or right arrow, it moves X pixels along (tested with Safari and Firefox). This means having to make multiple clicks to get to the next available tab. The [Release Notes](https://quasar.dev/start/release-notes) page on Quasar docs provides an example. To get to the position shown below, where I can just about click on the far right tab ('@Quasar/vite-plugin') I have to click the right arrow 10 times to gradually shift the tabs along to expose the last tab (using desktop). 

<img width="847" alt="Screenshot 2022-05-11 at 14 50 25" src="https://user-images.githubusercontent.com/64841595/167865828-5b7db10c-48ac-4c81-8c3f-e4bff08da98f.png">

It would be nice if there was an option so that 1 click on the right arrow scrolls across to expose the whole of the next tab (the fact I am scrolling right suggests that is what I am trying to achieve). 

The standard codepens from the docs provides another example:

https://quasar.dev/vue-components/tabs#outside-inside-and-visible-on-mobile-arrows

**Comment by ssuess** [comment]
I agree, a click should forward to the next full tab name. Also, would it be (or is it) possible to allow scrolling in the tabs if there is overflow like this? It would be great if (like with table overflow) one could just scroll to the next/previous tabs that are out of view

**Comment by ssuess** [comment]
Can we revisit this or does anyone have a solution? It is really bad ux for the user to have to click multiple times to scroll in this way (or hold down to get to the end quickly). Is there a workaround that would allow clicking to advance to the next tab?
